### PR TITLE
Document Proxy-Wasm ABI v0.2.0.

### PR DESCRIPTION
**Pull request by PiotrSikora**

### CODE REVIEW

**Comment by PiotrSikora** [comment]
This is pretty trivial update:
```
diff -u abi-versions/v0.2.{0,1}/README.md
```
```
--- abi-versions/v0.2.0/README.md       2024-08-07 16:13:04.836878636 -0400
+++ abi-versions/v0.2.1/README.md       2024-08-07 16:12:20.688649849 -0400
@@ -1,4 +1,4 @@
-# Proxy-Wasm ABI v0.2.0 specification
+# Proxy-Wasm ABI v0.2.1 specification
 
 ---
 
@@ -53,7 +53,7 @@
 
 ### Callbacks exposed by the Wasm module
 
-#### `proxy_abi_version_0_2_0`
+#### `proxy_abi_version_0_2_1`
 
 * params:
   - none
@@ -61,7 +61,7 @@
   - none
 
 Function marker used during linking to advertise Wasm module's support
-for Proxy-Wasm ABI v0.2.0.
+for Proxy-Wasm ABI v0.2.1.
 
 This function is never called.
 
@@ -333,6 +333,29 @@
   point to invalid memory address.
 
 
+#### `proxy_get_log_level`
+
+* params:
+  - `i32 (`[`proxy_log_level_t`]` *) return_log_level`
+* returns:
+  - `i32 (`[`proxy_status_t`]`) status`
+
+Retrieves host's current log level (`return_log_level`).
+
+This can be used to avoid creating log entries that are going to be
+discarded by the host.
+
+> **Note**
+> Hosts might change the log level at runtime, and currently there is
+> no callback to notify the Wasm module about it, so `return_log_level`
+> can become stale.
+
+Returned `status` value is:
+- `OK` on success.
+- `INVALID_MEMORY_ACCESS` when `return_log_level` points to invalid
+  memory address.
+
+
 ## Clocks
 
 ### Functions exposed by the host
@@ -2058,11 +2081,18 @@
 - `MONOTONIC` = `1`
 
 
+# Changes from [v0.2.0]
+
+- Added [`proxy_get_log_level`].
+
+
+[v0.2.0]: ../v0.2.0
+
 [integration]: #Integration
 [memory management]: #Memory-management
 [serialized]: #Serialization
 
-[`proxy_abi_version_0_2_0`]: #proxy_abi_version_0_2_0
+[`proxy_abi_version_0_2_1`]: #proxy_abi_version_0_2_1
 [`_initialize`]: #_initialize
 [`main`]: #main
 [`_start`]: #_start

```

File: abi-versions/v0.2.1/README.md [hunk 1]
```diff
@@ -2081,6 +2081,13 @@ changes to unrelated connections/requests.
 - `MONOTONIC` = `1`
 
 
+# Changes from [v0.2.0]
```
Comment (PiotrSikora) [inline abi-versions/v0.2.1/README.md:2084]: I'm not sure if this is best placed at the end of the beginning of the file. Happy to change it if someone has any preference.

**Review by PiotrSikora — COMMENTED** [review]
This is mostly a stepping stone for documenting v0.1.0, and kicking off process for v0.3.0.

**Review by mpwarres — APPROVED** [review]

File: abi-versions/v0.2.1/README.md (hunk 1 above)
Comment (mpwarres) [reply abi-versions/v0.2.1/README.md:2084]: Another option would be to have the changelog in spec/abi-versions/README.md

Comment (PiotrSikora) [reply abi-versions/v0.2.1/README.md:2084]: Good point. I think [CHANGELOG.md](https://keepachangelog.com/en/1.1.0/) (which we already use in Rust SDK) in the main directory might be even better.

Comment (mpwarres) [reply abi-versions/v0.2.1/README.md]: Agreed!

**Pull request merged**